flow/cmd/handler.go

```diff
@@ -156,31 +164,26 @@ func (h *FlowRequestHandler) CreateCDCFlow(
 	}, nil
 }
 
-func (h *FlowRequestHandler) updateFlowConfigInCatalog(
-	ctx context.Context,
-	cfg *protos.FlowConnectionConfigs,
-) error {
-	return internal.UpdateCDCConfigInCatalog(ctx, h.pool, slog.Default(), cfg)
-}
-
 func (h *FlowRequestHandler) CreateQRepFlow(
 	ctx context.Context, req *protos.CreateQRepFlowRequest,
 ) (*protos.CreateQRepFlowResponse, error) {
 	cfg := req.QrepConfig
-	cfg.Version = shared.InternalVersion_Latest
+	internalVersion, err := internal.PeerDBForceInternalVersion(ctx, req.QrepConfig.Env)
+	if err != nil {
+		return nil, err
+	}
+	cfg.Version = internalVersion
 
 	workflowID := fmt.Sprintf("%s-qrepflow-%s", cfg.FlowJobName, uuid.New())
 	workflowOptions := client.StartWorkflowOptions{
 		ID:                    workflowID,
 		TaskQueue:             h.peerflowTaskQueueID,
 		TypedSearchAttributes: shared.NewSearchAttributes(cfg.FlowJobName),
 	}
-	if req.CreateCatalogEntry {
```

Comment:
This one seems to be always set to true at call sites anyway. Should the proto field be removed or we might still want it?

Reply:
Trouble is backwards compatibility. Probably safe to remove since it's only request